dcgm/datadog_checks/dcgm/check.py

```diff
@@ -0,0 +1,28 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.base import OpenMetricsBaseCheckV2
+from datadog_checks.dcgm.metrics import METRIC_MAP
+
+
+class DcgmCheck(OpenMetricsBaseCheckV2):
+    __NAMESPACE__ = 'dcgm'
+    DEFAULT_METRIC_LIMIT = 0
+
+    def get_default_config(self):
+        return {
+            "metrics": [METRIC_MAP],
+        }
+
+    def configure_scrapers(self):
+        super().configure_scrapers()
+
+        self.scrapers[self.instance['openmetrics_endpoint']].metric_transformer.add_custom_transformer(
+            "build_information",
+            self._transform_build_information,
+        )
+
+    def _transform_build_information(self, metric, sample_data, runtime_data):
```

Comment:
I would suggest renaming this method to clearly communicate what it does. For instance: `_add_build_version_to_metadata` or `_include_build_version_in_metadata`.

Reply:
Done ✅